gradle/java.gradle

```diff
@@ -6,6 +6,10 @@ apply plugin: 'com.palantir.baseline-idea'
 
 sourceCompatibility = 1.8
 
+checkstyle {
+    configDir = rootProject.file(".baseline")
```

Comment:
Current baseline already sets configFile to `configDir/checkstyle/checkstyle.xml` and suppressions directory to `configDir/checkstyle`. My claim is when you release this and pick up here you would be able to remove this line